### PR TITLE
feat: use identities

### PR DESCRIPTION
**Pull request by jawid-h**

### CODE REVIEW

File: src/schema/dpns-documents.json [hunk 1]
```diff
@@ -1,77 +1,88 @@
...
-    "preorder": {
-        "indices": [
-            {
-                "properties": [
-                    { "$userId": "asc" },
-                    { "saltedDomainHash": "asc" }
-                ],
-                "unique": true
-            }
+        "unique": true
```
Comment (shumkov) [inline src/schema/dpns-documents.json]: Remove

File: src/schema/dpns-documents.json [hunk 2]
```diff
@@ -1,77 +1,88 @@
...
+      },
+      {
+        "properties": [
+          { "normalizedParentDomainName": "asc" }
         ],
+        "unique": true
```
Comment (shumkov) [inline src/schema/dpns-documents.json]: Remove

File: src/schema/dpns-documents.json [hunk 3]
```diff
@@ -1,77 +1,88 @@
...
-                    { "$userId": "asc" },
-                    { "saltedDomainHash": "asc" }
-                ],
-                "unique": true
-            }
+        "unique": true
+      },
+      {
+        "properties": [
+          { "normalizedParentDomainName": "asc" }
```
Comment (shumkov) [inline src/schema/dpns-documents.json]: We need a compound index with `normalizedParentDomainName`, `normalizedLabel `

**Review by shuplenkov — APPROVED** [review]
utACK

**Pull request merged**